interfaces/builtin/basedeclaration.go

```diff
@@ -68,6 +68,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  dcdbas-control:
+    allow-installation:
+      slot-snap-tupe:
+        - gadget
+    deny-auto-connection: true
```

Comment:
@jdstrand 

We originally intended to have the slot created in the gadget snap for Dell devices running core, however after discussion with Simon, this has been changed to an implicit core slot.  You're correct about there being no slot side support in the interface, which is because this is guarding a kernel-controlled service and there's no additional logic needed for a slot.